corehq/apps/app_manager/models.py

```diff
@@ -1795,10 +1795,10 @@ class DetailTab(IndexedSchema):
     # iterates through a set of entities rather than a single entity.
     # A nodeset is represented by one of the two properties:
     #   nodeset: An absolute xpath expression to iterate over
-    #   nodeset_child_case_type: Iterate over all child cases of this type
+    #   nodeset_case_type: Iterate over all child cases of this type
     has_nodeset = BooleanProperty(default=False)
     nodeset = StringProperty()
-    nodeset_child_case_type = StringProperty()
+    nodeset_case_type = StringProperty()
```

Comment:
I think the previous name `nodeset_child_case_type` was more clear though perhaps because this is in the context of a 'tab' it's OK.

Reply:
My thinking is that this work might expand beyond child cases to other relationships.